ui/src/message_popup/linear_session/linear_session.jsx

```diff
@@ -14,10 +14,22 @@ export default React.createClass({
   propTypes: {
     questions: React.PropTypes.array.isRequired,
     questionEl: React.PropTypes.func.isRequired,
+    getNextQuestion: React.PropTypes.func,
     summaryEl: React.PropTypes.func.isRequired,
     onLogMessage: React.PropTypes.func.isRequired
   },
 
+  getDefaultProps() {
```

Comment:
This is a small style suggestion, but if a "default method" gets more involved you can also split it out to be a method on the component and call it here, or a pure function at the top of the file.